fdbserver/workloads/SpecialKeySpaceCorrectness.actor.cpp

```diff
@@ -827,6 +827,10 @@ struct SpecialKeySpaceCorrectnessWorkload : TestWorkload {
 					ASSERT(schemaMatch(schema, valueObj, errorStr, SevError, true));
 					ASSERT(valueObj["command"].get_str() == "lock" && !valueObj["retriable"].get_bool());
 					break;
+				} else if (e.code() == error_code_database_locked) {
+					// Database is already locked. This can happen if a previous attempt
```

Comment:
Just curios to know, does the database means a cluster or a SS? Can this happen for any other reason other than the previous attempt with unknown_result?

Reply:
yeah, it means the whole cluster.
Almost it happens with `unknown_result`, which is the specific error where the commit may succeed but we do not know.